pkg/controller/garbagecollector/metrics.go

```diff
@@ -0,0 +1,113 @@
+/*
```

Comment:
You should add line similar to https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/replication/replication_controller.go#L122 to garbage_collector to monitor client usage.